store/file/ods.go

```diff
@@ -243,6 +284,14 @@ func (f *ODSFile) readAxisHalf(axisType rsmt2d.Axis, axisIdx int) (eds.AxisHalf,
 	return eds.AxisHalf{}, fmt.Errorf("unknown axis")
 }
 
+func (f *ODSFile) sharesOffset() int {
+	return f.hdr.Size() + f.axisRootsSize()
+}
+
+func (f *ODSFile) axisRootsSize() int {
+	return share.AxisRootSize * 2 * f.size()
```

Comment:
I would add where 2 is coming from.
```suggestion
	return share.AxisRootSize * 2 * f.size() // there are two Row and Col axises; thus we multiply by 2
```